AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/copy-conference-phone/copy-conference-phone.component.html

```diff
@@ -10,6 +10,6 @@
     (mouseout)="onMouseOut()"
     (click)="copyToClipboard(); $event.stopPropagation()"
   >
-    {{ _detailsToCopy }}
+    telephone-participant-link
```

Comment:
Just a question here, I am wondering as to why its been changed to a magic string? Looked like previously we were getting the value from a property defined in this component

Reply:
string were a bit lenghty so it was decided to use a generic value and the real clipboard value behind the scenes